### PR TITLE
unixodbc 2.3.4

### PR DESCRIPTION
**Pull request by tdsmith**
Closes https://github.com/Homebrew/legacy-homebrew/issues/31090.

### CODE REVIEW

**Comment by DomT4** [comment]
Forgot the mirror :wink:

**Comment by tdsmith** [comment]
Ugh, murder my face, I did

File: Formula/unixodbc.rb [hunk 1]
```diff
@@ -28,4 +27,8 @@ def install
                           "--enable-gui=no"
     system "make", "install"
   end
+
+  test do
+    system "odbcinst", "-j"
```
Comment (mikemcquaid) [inline Formula/unixodbc.rb]: `#{bin}` 😉

Comment (tdsmith) [inline Formula/unixodbc.rb]: fixed, thanks

**Comment by tdsmith** [comment]
Oops; closed by d140e37e99b1.

**Pull request closed without merging**